### PR TITLE
[SIG-4170] Pin location on object map

### PR DESCRIPTION
**Pull request by janjaap**
This PR:
- Enables pinning a location on the asset selector map; pinning a location will deselect the previously selected objects. Selecting an object will deselect the pinned location.
- Has a couple of more options to configure the labels in the asset selector
- Moves the reverse geocoder service to a more generic location in the file tree and adds a couple of missing types

### CODE REVIEW

File: src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/__tests__/WfsLayer.test.tsx [hunk 1]
```diff
@@ -39,7 +38,7 @@ const endpoint = 'https://endpoint/?version=2'
 const promise = Promise.resolve()
 const assetSelectProviderValue: AssetSelectValue = {
   selection: [],
-  location: new LatLng(0, 0),
+  location: [0, -0],
   meta: {
```
Comment (kiruys) [inline src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/__tests__/WfsLayer.test.tsx:42]: why do you use a negative 0 here?

File: src/signals/incident/components/form/MapSelectors/Asset/__tests__/AssetSelect.test.tsx [hunk 2]
```diff
@@ -89,4 +125,74 @@ describe('AssetSelect', () => {
...
+    expect(updateIncident).toHaveBeenLastCalledWith({
+      location: expect.objectContaining({
+        geometrie: expect.objectContaining({ type: 'Point' }),
+        address: geocodedResponse.data.address,
+      }),
+    })
+  })
+
+  it('handles setLocation for a unsuccessful reverse geocode request', async () => {
+    const geocodedResponse = undefined
```
Comment (kiruys) [inline src/signals/incident/components/form/MapSelectors/Asset/__tests__/AssetSelect.test.tsx:175]: when does this happen? what does it mean?

**Review by kiruys — APPROVED** [review]
👍

File: src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/__tests__/WfsLayer.test.tsx (hunk 1 above)
Comment (janjaap) [reply src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/__tests__/WfsLayer.test.tsx:42]: Hm, must've happened when I reformatted the whole file. Definitely wasn't intentional. Doesn't make a difference, tho.

File: src/signals/incident/components/form/MapSelectors/Asset/__tests__/AssetSelect.test.tsx (hunk 2 above)
Comment (janjaap) [reply src/signals/incident/components/form/MapSelectors/Asset/__tests__/AssetSelect.test.tsx:175]: It happens when the reverse geocoder cannot find an address from a GPS coordinate

**Pull request merged**